sormas-backend/src/main/resources/sql/sormas_schema.sql

```diff
@@ -6762,4 +6762,19 @@ ALTER TABLE facility ADD COLUMN areatype varchar(255);
 
 INSERT INTO schema_version (version_number, comment) VALUES (340, 'Add facilities'' address #4027');
 
+-- 2020-02-19 Add person email, person phone to LabMessage #4106
+ALTER TABLE labmessage ADD COLUMN personphone VARCHAR(255);
+ALTER TABLE labmessage ADD COLUMN personemail VARCHAR(255);
+ALTER TABLE labmessage_history ADD COLUMN personphone VARCHAR(255);
+ALTER TABLE labmessage_history ADD COLUMN personemail VARCHAR(255);
+
+INSERT INTO schema_version (version_number, comment) VALUES (341, 'Add person email, person phone to LabMessage #4106');
+
+-- 2020-03-04 Add testResultVerified to LabMessage #4106
+ALTER TABLE labmessage ADD COLUMN testresultverified boolean DEFAULT false;
+ALTER TABLE labmessage_history ADD COLUMN testresultverified boolean DEFAULT false;
+UPDATE labmessage SET testresultverified = true;
+
+INSERT INTO schema_version (version_number, comment) VALUES (342, 'Add testResultVerified to LabMessage #4106');
```

Comment:
Wouldn't it be better to have just one new entry in the schema_version? The only ones required to modify databases are you and me.

Reply:
Ok, I will concatenate them